auth-web/src/components/auth/account-settings/team-management/PendingMemberDataTable.vue

```diff
@@ -88,10 +87,6 @@ export default class PendingMemberDataTable extends Vue {
     }
   ]
 
-  private canApproveOrDeny (): boolean {
```

Comment:
can be consistent now with some of the current invitation updates to use permissions.